docs/api/mount.md

```diff
@@ -170,7 +170,7 @@ Simulates an event on the current node.
 #### [`.setState(nextState) => ReactWrapper`](ReactWrapper/setState.md)
 Manually sets state of the root component.
 
-#### [`.setProps(nextProps) => ReactWrapper`](ReactWrapper/setProps.md)
+#### [`.setProps(nextProps[, callback]) => ReactWrapper`](ReactWrapper/setProps.md)
```

Comment:
Thanks, does a similar change need to be made in the shallow docs?

Reply:
Without looking at any of the code, the documentation for shallow's `setProps` does not describe a callback argument.